spec/queries/registered_patients_query_spec.rb

```diff
@@ -12,12 +14,23 @@
       create(:patient, :without_hypertension, registration_facility: facility)
       create_list(:patient, 1, registration_facility: other_facility, registration_user: user2)
     end
-    result = RegisteredPatientsQuery.new.count(facility, :month)
+    result = described_class.new.count(facility, :month)
     expect(result[Period.month("December 1st 2020")]).to be_nil
     expect(result[Period.month("January 1st 2020")]).to eq(2)
     expect(result[Period.month("April 1st 2020")]).to eq(2)
   end
 
+  it "includes dead patients" do
```

Comment:
@claudiovallejo @prabhanshuguptagit verifying this is true for the registered patients query.